src/public/modules/core/services/gtag.client.service.js

```diff
@@ -4,6 +4,7 @@ function GTag($rootScope, $window) {
   // Google Analytics tracking ID provided on signup.
   const GATrackingID = $window.GATrackingID
   let gtagService = {}
+  const userEmail = JSON.parse($window.localStorage.getItem("user")).email
```

Comment:
I think tests are failing because this is your `JSON.parse` returns undefined at the time the service is instantiated, resulting in a null pointer exception. Should probably shift this code within the function and maybe include some type safety with `_.get`.